CODIGO/clsGuildDlg.cls

```diff
@@ -317,3 +318,14 @@ font_Err:
     Resume Next
 End Property
 
+' Toggle visibility of guild dialogs when user opens guild chatbox
+Public Sub toggle_dialogs_visibility(ByVal visible As Boolean)
+    ' Author: Alexis Caraballo (WyroX)
+    ' Last Modify Date: 30/07/2022
```

Comment:
Please remove for the sake of brevity.
Imagine what would happen if everybody adds his name.

The list of contributors would should your name.